rpcserver.go

```diff
@@ -1437,8 +1449,8 @@ func (r *rpcServer) SendCoins(ctx context.Context,
 		paymentMap := map[string]int64{targetAddr.String(): in.Amount}
 		err := wallet.WithCoinSelectLock(func() error {
 			newTXID, err := r.sendCoinsOnChain(
-				paymentMap, feePerKw, minConfs,
-				label, coinSelectionStrategy,
+				paymentMap, feePerKw, minConfs, label,
```

Comment:
the `paymentMap` contains amounts that are used in `sendCoinsOnChain` to create outputs with `addrPairsToOutputs`. If we select utxos manually, do we have to change the amount in the `paymentMap`?

Reply:
No, I do not think so. Why would we want to do that?

Reply:
I thought cause the outputs are created in `sendCoinsOnChain` based on the payment map. How does this behave if utxos are selected and `--sweepall` as well. Shouldn't the payment map contain the sum of the value of the selected utxos?

Reply:
when used with sweepall, it does not execute the else branch that leads to a call to `sendcoinsonchain`. Rather the `CraftSweepAllTx` is what is invoked. I did that because the CraftSweepAllTx already has everything in place to send all outputs in select utxos as change outputs, adjusting only if a particular amount should be reserved.